app/commands.py

```diff
@@ -60,3 +63,19 @@ def run(self, service_name_prefix=None, user_email_prefix=None):
                     else:
                         delete_user_verify_codes(usr)
                         delete_model_user(usr)
+
+
+class CustomDbScript(Command):
+    def run(self):
+        self.update_notification_international_flag()
```

Comment:
should look at how to give this an argument, so you call `python application.py custom_db_script <name_of_function>` rather than us having to update this the next time we run stuff